app/com/arpnetworking/metrics/portal/reports/impl/chrome/OriginConfig.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2019 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.arpnetworking.metrics.portal.reports.impl.chrome;
+
+import com.arpnetworking.commons.builder.OvalBuilder;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+/**
+ * Describes special actions that should be done for some origin (i.e. scheme+host+port).
+ *
+ * @author Spencer Pearson (spencerpearson at dropbox dot com)
+ */
+@JsonDeserialize(builder = OriginConfig.Builder.class)
```

Comment:
This should not be necessary if you are serializing/deserializing via the injected ObjectMapper.